src/main/java/com/synopsys/integration/detect/ApplicationUpdater.java

```diff
@@ -286,6 +285,12 @@ private boolean runMainClass(Path jarPath)
         return true;
     }
     
+    @java.lang.SuppressWarnings("java:S3011")
```

Comment:
The warning is suppressed.
Note: It was not possible to confirm if this will indeed suppress the warning on Sonar CF - that will be confirmed once the PR is merged to master.